content/integrations/streaming/bigquery.textile

```diff
@@ -0,0 +1,109 @@
+---
+title: Google BigQuery
+meta_description: "Stream realtime event data from Ably into Google BigQuery using the Firehose BigQuery rule. Configure, and analyze your data efficiently."
+---
+
+Stream events published to Ably directly into a "table":https://cloud.google.com/bigquery/docs/tables in "BigQuery":https://cloud.google.com/bigquery for analytical or archival purposes. General use cases include:
+
+* Realtime analytics on message data.
+* Centralized storage for raw event data, enabling downstream processing.
+* Historical auditing of messages.
+
+To stream data from Ably into BigQuery, you need to create a BigQuery "rule":#rule.
+
+<aside data-type='note'>
+<p>Ably's BigQuery integration for "Firehose":/docs/integrations/streaming is in alpha status.</p>
+</aside>
+
+h2(#rule). Create a BigQuery rule
+
+A rule defines what data gets sent, where it goes, and how it's authenticated. For example, you can improve query performance by configuring a rule to stream data from a specific channel and write them into a "partitioned":https://cloud.google.com/bigquery/docs/partitioned-tables table.
+
+h3(#dashboard). Create a rule using the Ably dashboard
+
+The following steps to create a BigQuery rule using the Ably dashboard:
+
+* Log in to the "Ably dashboard":https://ably.com/accounts/any and select the application you want to stream data from.
+* Navigate to the *Integrations* tab.
+* Click *New integration rule*.
+* Select *Firehose*.
+* Choose *BigQuery* from the list of available Firehose integrations.
+* "Configure":#configure the rule settings. Then, click *Create*.
+
+h3(#api-rule). Create a rule using the ABly Control API
+
+The following steps to create a BigQuery rule using the Control API:
```

Comment:
This doesn't exist in the Control API spec at the moment, so I think we need to check whether this is possible.